test/integration/controllers/v1/ApiController.test.coffee

```diff
@@ -0,0 +1,62 @@
+###*
+# ApiController.test
+###
+
+request = require('supertest')
+should = require('should')
+
+###*
+# Test API "ping" endpoint.
+###
+
+describe 'Requests to the root (/api) path', ->
+  it 'GET: Returns a 200 status code', (done) ->
+    request(sails.hooks.http.app)
+      .get('/api')
```

Comment:
You don't need 2 spaces indentation before get and expect. See simpler chaining syntax example in notes to CoffeeScript 1.7.0: http://coffeescript.org/#1.7.0.
Also, parentheses are optional.